modules/oneVideoBidAdapter.js

```diff
@@ -302,6 +302,14 @@ function getRequestData(bid, consentData, bidRequest) {
     bidData.site.ref = 'https://verizonmedia.com';
     bidData.tmax = 1000;
   }
+  if (bid.params.video.custom && Object.prototype.toString.call(bid.params.video.custom) === '[object Object]') {
```

Comment:
I think we can use `typeof` check for object too.
i.e., use `typeof bid.params.video.custom` instead of 
`Object.prototype.toString.call(bid.params.video.custom) === '[object Object]'`

![Screenshot 2020-12-01 at 9 16 09 PM](https://user-images.githubusercontent.com/10308692/100762840-73b57d00-341a-11eb-8017-648f566c4a12.png)

Reply:
oh! you are checking for skipping array?

Reply:
I tried that before and found that `typeof` fails when passing: Array.
Then discovered that it also fails when passing Set, Date & Null - they are all considered objects.
`Object.prototype.toString.call(bid.params.video.custom) === '[object Object]'` ensures that you can only pass an Object
and works when validating: 
- String
- Object
- Array
- Set
- Date
- Undefined
- Null
If this is overkill let me know - but I wanted to make sure nothing else is passed.

Reply:
I think what you are doing is best for skipping other type values. 👍🏼

Reply:
I saw this discussion, I just want to share my advice, this is for future PRs:  
it would be easier/shorter just to execute `utils.isPlainObject(bid.params.video.custom)`.

Reply:
Just went through the `utils`,  Thank you @FilipStamenkovic there are lot of helpers there will use them going forward.

Reply:
Thank you so much @FilipStamenkovic that is perfect! 
I will update accordingly